docs/src/introduction/overview.md

```diff
@@ -0,0 +1,229 @@
+# Getting Started
+
+Follow this guide to write and deploy a simple wallet smart contract in Sway.
+
+## Glossary
+
+Before we begin, it may be helpful to understand terminology that will used throughout the docs and how they relate to each other:
+
+- **Fuel**: the Fuel blockchain.
+- **FuelVM**: the virtual machine powering Fuel.
+- **Sway**: the domain-specific language crafted for the FuelVM; it is inspired by Rust.
+- **Forc**: the build system and package manager for Sway, similar to Cargo for Rust.
+
+Go [here](./sway-toolchain.md) for more detailed information about each of these components.
+
+## Understand Sway Program Types
+
+There are four types of Sway programs:
+
+- contract
+- predicate
+- script
+- library
+
+Contracts, predicates, and scripts can produce artifacts usable on the blockchain, while a library is simply a project designed for code reuse and is not directly deployable.
+
+Every Sway file must begin with a declaration of what type of program it is.
+
+## Create Wallet Projects with `forc`
+
+To deploy a wallet on Fuel, we will need to write a library, a contract, and a script in Sway.
+
+First, let's [install the Sway toolchain](./installation.md). Then with `forc` installed, let's create three different sibling projects:
+
+```console
```

Comment:
Here and in other place:

1. Use `plaintext` if it's just text (not often).
2. Use `console` if there's a `$` in front of commands followed by output.
3. Use `sh` if it's commands with no output and no `$`.